core/vm/contracts.go

```diff
@@ -141,9 +141,9 @@ var PrecompiledContractsBerlin = map[common.Address]PrecompiledContract{
 	common.BytesToAddress([]byte{9}): &blake2F{},
 }
 
-// PrecompiledContractsBoneh contains the default set of pre-compiled Ethereum
+// PrecompiledContractsLynn contains the default set of pre-compiled Ethereum
 // contracts used in the Boneh release.
-var PrecompiledContractsBoneh = map[common.Address]PrecompiledContract{
```

Comment:
@KeefeL do have a precompile contract that want to enable at Boneh, so such modification may conflict with his intension.